src/debootstrap/tune-wget-settings.patch

```diff
@@ -0,0 +1,33 @@
+From a056a2d8cec783002b13c3592846e83ea1e9b5a2 Mon Sep 17 00:00:00 2001
+From: Konstantin Vasin <k.vasin@yadro.com>
+Date: Sun, 20 Nov 2022 19:22:59 +0300
+Subject: [PATCH] tune wget settings
+
+1. Set number of tries to 3 (default is 20).
+2. Set the connect timeout to 10 seconds
+(default value for Linux is 127 sec if tcp_syn_retries == 6).
+3. Set the read (and write) timeout to 15 seconds.
```

Comment:
Does it have impact on the users who have very bad network, for the first try? For example, very long latency, it can download the file with long time, but retry not help. It will change the default options only on the retry attempts (the second retry and the third retry), or all steps including the first attempt?

Reply:
It changes 1st attempt too.
When I tested with deb.debian.org most fails were like this:

`Resolving deb.debian.org (deb.debian.org)... 199.232.138.132, 2a04:4e42:3a::644
Connecting to deb.debian.org (deb.debian.org)|199.232.138.132|:80...`
After that it's waiting 127 sec and fail or try again if --retry-connrefused is set.

Also one time I got read error:
`Resolving deb.debian.org (deb.debian.org)... 199.232.138.132, 2a04:4e42:3a::644
Connecting to deb.debian.org (deb.debian.org)|199.232.138.132|:80... connected.
HTTP request sent, awaiting response...`
After that it's waiting 900 sec and try again.

So I reduced this timeouts to spedup build.
But ofc I can do additional tests using  [netem](https://wiki.linuxfoundation.org/networking/netem)
It allows to simulate bad network connection and customize latency, rate and packet loss.

Reply:
Results with bad network connection were more stable than high bandwidth network connection.
Maybe this error is some sort of "rate limit" of public debian servers.

Also I have tested another public debian mirror.
And unlike deb.debian.org it sometimes responses me after 60-100 sec connection timeout.
So I decided to remove both these options: --connect-timeout and --read-timeout and use default values.
I'm still sure  that short values are better for more faster build if we have a stable network connection.
But as a generic solution default values of timeouts are acceptable.